projects/smoldot-extension/src/components/AntSwitch.tsx

```diff
@@ -0,0 +1,70 @@
+import React, { FunctionComponent, useState } from 'react';
+import { Switch } from '@material-ui/core';
+import { withStyles, createStyles, Theme } from '@material-ui/core/styles';
+
+interface Props {
+    isActive?: boolean;
+    activeColor?: string;
+    activeBgColor?: string;
+    activeBorderColor?: string;    
+    deactiveColor?: string;
+    deactiveBgColor?: string;
+    deactiveBorderColor?: string;
+    size?: string;
+}
+
+const AntSwitch: FunctionComponent<Props> = ({
```

Comment:
Why is this called "AntSwitch" out of interest?  Is this a special term?

Reply:
I think they gave from material UI this name cause it's design is very similar to AntDesign. I kept it as of tutorials.